### PR TITLE
Wyndex migration/implement deposit freeze

### PR DESCRIPTION
**Pull request by ueco-jb**
Freeze deposits on junoswap

### CODE REVIEW

File: src/contract.rs [hunk 1]
```diff
@@ -185,9 +208,23 @@ pub fn execute(
             protocol_fee_percent,
             protocol_fee_recipient,
         ),
+        ExecuteMsg::FreezeDeposits {} => execute_freeze_deposits(deps, info.sender),
     }
 }
 
+fn execute_freeze_deposits(deps: DepsMut, sender: Addr) -> Result<Response, ContractError> {
```
Comment (ben2x4) [inline src/contract.rs:215]: i think there should be an unfreeze option? Maybe pass a pool with this message?

**Review by ben2x4 — CHANGES_REQUESTED** [review]
LGTM, just one small comment for unfreezing a pool for completeness

File: src/contract.rs (hunk 1 above)
Comment (ueco-jb) [reply src/contract.rs]: I added boolean in the message.

**Review by ben2x4 — APPROVED** [review]
LGTM

**Pull request merged**